src/components/Sidebar/LeftSidebar.tsx

```
@@ -0,0 +1,187 @@
+  ][];
+
+  useEffect(() => {
+    const activeProduct = activePageHierarchy[0];
+    if (!selectedProduct && activeProduct !== undefined) {
+      setSelectedProduct(products[activeProduct][0]);
+    }
+  }, [activePageHierarchy, products, selectedProduct]);
+
+  useEffect(() => {
+    if (selectedLinkId) {
+      const element = typeof document !== `undefined` ? document.getElementById(selectedLinkId) : null;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)